### PR TITLE
Polish and improve track and change for new token

### PR DESCRIPTION
**Pull request by icerove**
1. update tracking with new design and small issues
2. change token to new one

### CODE REVIEW

**Comment by icerove** [comment]
Please review and fix and merge this before release. @vgrichina 
@Patrick1904

File: src/mixpanel/index.js [hunk 1]
```diff
@@ -9,8 +9,16 @@ let Mixpanel = {
         set: () => {},
         set_once: ()  => {}
     },
-    withTracking: () => {}
-};
+    withTracking: async (name, fn, errorOpration = () => {}, finalOperation = () => {}) => {
```
Comment (vgrichina) [inline src/mixpanel/index.js:12]: nit: typo `errorOpration` instead of `errorOperation`

**Comment by vgrichina** [comment]
@ihaid please help testing if this doesn't break anything, especially 2FA

**Comment by icerove** [comment]
testing with environment variable and without that, all test are okay

**Pull request merged**